CMakeLists.txt

```diff
@@ -19,6 +20,12 @@ endif(NOT WIN32)
 file(READ "VERSION" PROJECT_FILE_VERSION)
 project(Griffeye.VideoProcessor.Native VERSION ${PROJECT_FILE_VERSION} DESCRIPTION "Video frame extraction library using ffmpeg")
 
+option(BUILD_EXAMPLES "build examples" OFF)
+
+if(CMAKE_BUILD_TYPE STREQUAL "")
+    set(CMAKE_BUILD_TYPE "Debug")
```

Comment:
Building from the command line was not working (`cmake -S . -B build_linux/`) since CMAKE_BUILD_TYPE was not set at that stage. That mean that the ffmpeg packages couldn't be found because the path was incomplete.